Lib/ufo2ft/outlineCompiler.py

```diff
@@ -410,6 +410,134 @@ def setupTable_cmap(self):
             # update tables registry
             cmap.tables = [cmap4_0_3, cmap4_3_1, cmap12_0_4, cmap12_3_10]
 
+    def _calcCodepageRanges(self):
+        codepageRanges = set()
+
+        unicodes = [i for i in self.unicodeToGlyphNameMapping.keys() if i is not None]
```

Comment:
the keys in that mapping are never None, you don't need to check `if i is not None`. If we do checks like that elsewhere, we need to change it.

Reply:
Yes, I copied that from somewhere else in the same file.